lib/purescript-cst/src/Language/PureScript/CST/Errors.hs

```diff
@@ -193,3 +194,5 @@ prettyPrintWarningMessage ParserErrorInfo {..} = case errType of
     "Kind imports are deprecated and will be removed in a future release. Omit the 'kind' keyword instead."
   WarnDeprecatedKindExportSyntax ->
     "Kind exports are deprecated and will be removed in a future release. Omit the 'kind' keyword instead."
+  WarnDeprecatedCaseOfOffsideSyntax ->
+    "An expression that is not indented past its `case ... of` branch's binder is deprecated and will be removed in a future release. Indent the expression beyond its binder, similar to how let bindings work."
```

Comment:
I don't think we need to reference let. Maybe something like:
```
Dedented expressions in case branches are deprecated and will be removed in a future release. Indent the branch's expression past it's binder instead.
```
It's a tough concept to word succinctly.

Reply:
Warning has been updated to use the above rendering. Thanks!